### PR TITLE
Add upgrades workload to PerfScale CI

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds support to upgrade the OpenShift cluster to a
desired build or the latest build given a upgrade path or channel
to use.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
Depends on https://github.com/cloud-bulldozer/benchmark-wrapper/pull/258.

**Review by jtaleric — COMMENTED** [review]
Stating `Please set this only` makes it sound optional, is that optional?

**Comment by chaitanyaenr** [comment]
@jtaleric yes they are optional and are only valid when the pipeline mode is enabled which runs a well defined set of stages in a particular order.

File: Jenkinsfile [hunk 1]
```diff
@@ -185,13 +186,31 @@ node (node_label) {
...
+
+		// Run upgrades
+		if ( upgrades == "TRUE" ) {
+			env.PIPELINE_STAGE="upgrades"
+			env.WORKLOAD_PROPERTIES_FILE=OPENSHIFTv4_SCALE_PROPERTY_FILE
+			env.WORKLOAD="ATS-SCALE-CI-SCALE"
+			load "pipeline-scripts/workload.groovy"
+                        env.WORKLOAD_PROPERTIES_FILE=CLUSTER_DENSITY_PROPERTIES_FILE
+                        env.WORKLOAD="RIPSAW-CLUSTER-DENSITY"
+                        load "pipeline-scripts/workload.groovy"
```
Comment (mohit-sheth) [inline Jenkinsfile:199]: spaces tabs issue

Comment (chaitanyaenr) [reply Jenkinsfile]: Fixed it, thanks.

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Pull request closed without merging**